portal/jam-embedded-api.md

```diff
@@ -92,180 +92,83 @@ The `share` method accepts an optional argument of the `ShareOptions` object typ
-                captureConsole: true,
-                reloadPage: false
-            };
-
-            function getCaptureInfo(jamState) {
-                switch (jamState) {
-                    case 'ready':
-                    case 'initialized':
-                        return 'Ready to capture..';
-                    case 'starting':
-                        return 'Starting..';
-                    case 'started':
-                        return 'Started..';
-                    case 'stopped':
-                        return 'Stopped..';
-                    case 'sharing':
-                        return 'Sharing..';
-                    case 'shared':
-                        return 'Sharing complete! Jam Session Url copied to clipboard.';
-                    default:
-                        return '';
-                }
-            }
-
-            function updateCaptureInfo() {
-                captureInfo = getCaptureInfo(jam.state);
-                document.getElementById('capture-info').innerHTML = captureInfo;
-
-                if (jam.options) {
-                    captureOptions = jam.options;
-                }
-
-                document.getElementById('captureVideo').checked = captureOptions.captureVideo;
-                document.getElementById('captureScreenshots').checked = captureOptions.captureScreenshots;
-                document.getElementById('captureStorage').checked = captureOptions.captureStorage;
-                document.getElementById('captureLogs').checked = captureOptions.captureConsole;
-            }
-
-            function initSettingsEvents() {
-                document.getElementById('captureVideo').addEventListener('change', (e) => {
-                    captureOptions.captureVideo = e.target.checked;
-                });
-
-                document.getElementById('captureScreenshots').addEventListener('change', (e) => {
-                    captureOptions.captureScreenshots = e.target.checked;
-                });
-
-                document.getElementById('captureStorage').addEventListener('change', (e) => {
-                    captureOptions.captureStorage = e.target.checked;
-                });
-
-                document.getElementById('captureLogs').addEventListener('change', (e) => {
-                    captureOptions.captureConsole = e.target.checked;
-                });
-            }
-
-            window.addEventListener('load', () => {
-                updateCaptureInfo();
-
-                jam.addStateChangedEventListener(newState => {
-                    updateCaptureInfo();
-                });
-
-                initSettingsEvents();
+            const jamEmbeddedScript = document.getElementById('jamEmbeddedScript');
+            jamEmbeddedScript.crossOrigin = 'anonymous'; // set crossOrigin to enable CORS (and to be able to use the load event below)
+            jamEmbeddedScript.addEventListener('load', () => {
+                // Adding the custom implmenetaion
```

Comment:
implmenetaion -> implementation

Reply:
Also, we can add a comment that they can place the Jam Embedded implementation here as well, but we recommend them to use a separate file for easier maintenance